examples/open_contracts/index.js

```diff
@@ -0,0 +1,41 @@
+import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api/dist/DerivAPIBasic";
+
+const app_id = 32486; // Replace with your app_id or leave as 1089 for testing.
```

Comment:
Lets update this comment. It says "leave as 1089" even though we use different app_id.